### DISCUSSION TITLE
how to implement the 'like' bool expression using expr

### DISCUSSION THREAD

**Discussion by cocktail828** [antonmedv/expr#681] [Q&A]
hello maintainers. I am trying to parse bool expression like this using expr `a>10 && b like '%abc' && c not in ['a', 'b']`.
But expr does not support `like` natively. So is there any ways to make expr support 'like' bool expression.
Thanks, in advance!

**Comment by nikolaymatrosov** [comment]
The `matches` operator is likely to meet your requirements. It verifies whether the left-hand operand conforms to the regular expression specified by the right-hand operand. For instance:

```
"aaa" matches "^a+$" == true
"abc" matches "^a+$" == false
```

https://expr-lang.org/docs/language-definition#operators
